tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/RecordSession.cs

```diff
@@ -102,9 +102,6 @@ public RecordEntry Lookup(RecordEntry requestEntry, RecordMatcher matcher, IEnum
             {
                 sanitizer.Sanitize(requestEntry);
             }
-            // normalize request body with STJ using relaxed escaping to match behavior when Deserializing from session files
```

Comment:
We no longer need to normalize bytes because if it's JSON, we compare by value now.

Reply:
We now need to add this back now that we check the sequenceequal FIRST. the normalization still needs to happen to make certain request matches response.